### ISSUE TITLE
ITMS-90809: Deprecated API Usage  Workaround in docs doesn't work

### ISSUE THREAD

**Issue by BrianJSavage** [MicrosoftDocs/xamarin-docs#2797]
A previously working version of an app in development used to be accepted with the deprecation message, but after May 1st 2020 this no longer appears to be the case.

I have attempted every single combination of the options listed in the docs which describe a workaround and nothing seems to get around the deprecation problem.

The image below is the last version I tried, but I have combined 'link all' with 'sdk only' and various versions of the top line configuration options.

![image](https://user-images.githubusercontent.com/40817356/80963406-f7968f00-8e62-11ea-984e-60dee548efdd.png)

I noticed that references to third party software may break this workaround, below are the packages the app uses
![image](https://user-images.githubusercontent.com/40817356/80963656-899e9780-8e63-11ea-8f00-367fe40aa75f.png)

---
#### Document details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: ef31dc71-3610-6c18-2bda-d0bb70804c92
* Version Independent ID: 02f7dc7b-541b-4b82-b33c-4e974fb94d8c
* Content: [Xamarin.Forms WebView - Xamarin](https://docs.microsoft.com/en-gb/xamarin/xamarin-forms/user-interface/webview?tabs=windows&source=docs#uiwebview-deprecation-and-app-store-rejection-itms-90809)
* Content Source: [docs/xamarin-forms/user-interface/webview.md](https://github.com/MicrosoftDocs/xamarin-docs/blob/live/docs/xamarin-forms/user-interface/webview.md)
* Product: **xamarin**
* Technology: **xamarin-forms**
* GitHub Login: @davidbritch
* Microsoft Alias: **dabritch**

**Comment by davidbritch** [comment]
Hi @BrianJSavage 

Take a look at https://docs.microsoft.com/xamarin/ios/user-interface/controls/webview#uiwebview-deprecation and let me know if anything there helps.

**Comment by BrianJSavage** [comment]
Saved by the bell, so to speak....
By chance I watched the latest from Channel 9:
https://channel9.msdn.com/Shows/XamarinShow/Six-Things-to-Love-in-XamarinForms-46--The-Xamarin-Show
and at about 6 mins in David Ortinau announced that in 4.6 they had sorted this issue.
So I upgraded to 4.6 and rebuilt the app and Apple accepted it. That saved me hours of messing around.
I left in --optimize=experimental-xforms-product-type in the mtouch arguments, next build I will try it without.
@davidbritch Thanks for the very prompt response, I'll close the issue but worth leaving up to help anyone else with the same problem.
